src/middleware/src/storage/src/indexeddb.js

```diff
@@ -29,7 +29,11 @@ export default class IndexedDB {
   }
 
   openTransaction(collection, write = false, success, error, force = false) {
-    const indexedDB = global.indexedDB || global.webkitIndexedDB || global.mozIndexedDB || global.msIndexedDB;
+    const indexedDB = global.indexedDB
+      || global.webkitIndexedDB
+      || global.mozIndexedDB
+      || global.msIndexedDB
+      || global.shimIndexedDB;
     let db = dbCache[this.name];
```

Comment:
@thomasconner This is more of a question than anything, but does it make sense to consolidate all of these DB checks into a single method?  Only asking because it looks like the same assignment is being made in a few different places, and having it all in one method may make it easier to change in the future.